features/kafka/producer/src/main/proto/opennms-kafka-producer.proto

```diff
@@ -73,6 +73,22 @@ message Alarm {
   repeated Alarm relatedAlarm = 22;
 }
 
+message AlarmFeedback {
```

Comment:
I think we were using underscore in naming fields in `.proto` file instead of camel case.

Reply:
Yep, you are right. I'll update that proto and regenerate the model file.